README.md

```diff
@@ -17,20 +17,33 @@ The currently supported functionality includes:
 - Deleting all Elastic IPs in an AWS account
 - Deleting all Launch Configurations in an AWS account
 - Deleting all ECS services in an AWS account
+- Deleting all ECS clusters in an AWS account
 - Deleting all EKS clusters in an AWS account
 - Deleting all RDS DB instances in an AWS account
 - Deleting all S3 buckets in an AWS account - except for buckets tagged with Key=cloud-nuke-excluded Value=true
 - Deleting all default VPCs in an AWS account
 - Revoking the default rules in the un-deletable default security group of a VPC
 
 ### Caveats
+- As of 2018-2019 AWS APIs seem to support tagging of ECS clusters, and therefore we've reviewed and re-focused on the support of nuking ECS clusters. 
```

Comment:
Thx for updating! Once this PR is ready to merge, all these ECS cluster caveats can be removed!